src/components/media.jsx

```diff
@@ -307,7 +307,7 @@ function Media({
                 }}
                 onError={(e) => {
                   const { src } = e.target;
-                  if (src === mediaURL) {
+                  if (src === mediaURL && mediaURL != remoteMediaURL) {
```

Comment:
Would be great if you can change this and above line to use `!==` instead. Thanks

Reply:
Updated :)